mne/io/boxy/boxy.py

```diff
@@ -527,6 +533,90 @@ def _read_segment_file(self, data, idx, fi, start, stop, cals, mult):
     
                                 # save our data based on data type
                                 data_[index_loc, :] = boxy_array[:, channel]
+                                
+                    ###phase unwrapping###
+                    # thresh = 0.00000001
+                    # scipy.io.savemat(file_name = r"C:\Users\spork\Desktop\data_matlab.mat",
+                    #                   mdict=dict(data=data_))
+                    if i_data == 'Ph':
+                        print('Fixing phase wrap')
+                        for i_chan in range(np.size(data_, axis=0)):
+                            if np.mean(data_[i_chan,:50]) < 180:
```

Comment:
can you explain what this does in english, and add a comment above with that

Reply:
I'm assuming it's done to account for instant changes in phase (crossing over 0/360 degrees)?

Not sure why it only considers the first 50 points though. Comments in ppod don't say why.

Reply:
its the mean of the first 50 points, funny for sure, a good question for Ed and gabriele if we meet tomorrow with them